### PR TITLE
New 7260cx3 deployment template

### PR DESCRIPTION
**Pull request by vaibhavhd**
### Description of PR

Summary: New 7260cx3 switch deployment template

### Type of change

- [ ] Bug fix
- [x] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

#### What is the motivation for this PR?
Adding new ansible template for 7260cx3 device

#### How did you verify/test it?
Successfully deployed configuration for a lab fanout switch using this template

#### Any platform specific information?
For Arista 7260cx3 device

### CODE REVIEW

**Comment by vaibhavhd** [comment]
retest vsimage please

File: ansible/roles/fanout/templates/arista_7260cx3_deploy.j2 [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+   speed forced 50gfull
+   no error-correction encoding
+   no shutdown
+{%         else %}
+   shutdown
+{%         endif %}
+{%     else %}
+{#        Not valid speed value #}
+   shutdown
+{%     endif %}
```
Comment (neethajohn) [inline ansible/roles/fanout/templates/arista_7260cx3_deploy.j2:90]: add ! here. we might not configure this intf properly if this section of code is hit. '!' is used to separate out each config chunk. if its missed, then the config will get overwritten with the next chunk

Comment (vaibhavhd) [reply ansible/roles/fanout/templates/arista_7260cx3_deploy.j2:90]: I tried to read this conditionals carefully. It seems like this `else` part's `if` statement already has a `!` before it..
```
!
if {}
elif {}
else {}
!
```

Can you double check if it is still needed?

Comment (neethajohn) [reply ansible/roles/fanout/templates/arista_7260cx3_deploy.j2:90]: I see that covered in 108. All good

**Review by neethajohn — APPROVED** [review]

**Pull request merged**